packages/fx-core/src/plugins/resource/function/utils/depsChecker/backendExtensionsInstall.ts

```diff
@@ -42,20 +42,28 @@ export class BackendExtensionsInstaller {
     }
 
     try {
-      await cpUtils.executeCommand(
-        backendRoot,
-        this._logger,
-        { shell: false },
-        dotnetExecPath,
-        "build",
-        csprojPath,
-        "-o",
-        outputPath,
-        "--ignore-failed-sources"
+      const result: cpUtils.ICommandResult = await cpUtils.tryExecuteCommand(
+          backendRoot,
+          this._logger,
+          { shell: false },
+          dotnetExecPath,
+          "build",
+          csprojPath,
+          "-o",
+          outputPath,
+          "--ignore-failed-sources"
       );
+      if (result.code !== 0) {
+        throw new Error(`Failed to run "${dotnetExecPath} build" command. output = ${result.cmdOutput}, err = ${result.cmdOutputIncludingStderr}`);
```

Comment:
Will this error be wrapped in telemetry? What if result contains user information?